aiida_zeopp/workflows/geometry.py

```diff
@@ -0,0 +1,112 @@
+from aiida.orm import CalculationFactory, DataFactory
+from aiida.orm.code import Code
+from aiida.orm.data.base import Float
+from aiida.work.run import submit
+from aiida.work.workchain import WorkChain, ToContext, if_, Outputs
+
+# import base data classes
+ZeoppCalculation = CalculationFactory('zeopp.network')
+ParameterData = DataFactory('parameter')
+CifData = DataFactory('cif')
+SinglefileData = DataFactory('singlefile')
+
+options = {
+    "resources": {
+        "num_machines": 1,
+        "tot_num_mpiprocs": 1,
+    },
+    "max_wallclock_seconds": 30 * 60,
+    "withmpi": False,
+}
+
+class ZeoppGeometryWorkChain(WorkChain):
```

Comment:
```suggestion
class ZeoppBlockedPocketsWorkChain(WorkChain):
```
"Geometry" is very generic (almost everything that zeo++ does is comute geometric properties).
What about "BlockedPockets"?